addons/website_event_exhibitor/models/ir_http.py

```diff
@@ -0,0 +1,20 @@
+from odoo import models
+
+
+class IrHttp(models.AbstractModel):
+    _inherit = 'ir.http'
+
+    def _get_robots_directives(self):
+        config = super()._get_robots_directives()
+
+        disallow_patterns = [
+            '/event/*/exhibitors?countries=',
+            '/event/*/exhibitors?*&countries=',
```

Comment:
Doesn't this cover all cases? (and other similar)
```suggestion
            '/event/*/exhibitors?*countries=',
```